chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -120,7 +120,7 @@
   group "root"
   action :create
   variables(
-    needs_block_migrate: nova[:nova][:use_shared_instance_storage] ? false : true
+    needs_block_migrate: nova.nil? || (nova[:nova][:use_shared_instance_storage] ? false : true)
```

Comment:
And as for `!nova[:nova][:use_shared_instance_storage]`, seems to make sense, but I'd rather not change the part that is working